File: src/test/java/util/DateUtils.java
package util;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;
import java.util.Locale;

public class DateUtils {

    private static final DateTimeFormatter dataFormatter = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
    private static final DateTimeFormatter dataTimeFormatter = DateTimeFormatter.ofPattern("d MMM yyyy HH:mm O", Locale.ENGLISH);

    public static LocalDate convertDate(String strDate) {
        return LocalDate.parse(strDate, dataFormatter);
    }

    public static LocalDateTime convertDateTime(String strDate) {
        return LocalDateTime.parse(strDate, dataTimeFormatter);
    }
}

File: src/test/kotlin/org/skefir/util/DateUtils.kt
package org.skefir.util

import java.time.LocalDate
import java.time.LocalDateTime
import java.time.format.DateTimeFormatter
import java.util.*


private val dataFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH)
    private val dataTimeFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMM yyyy HH:mm O", Locale.ENGLISH)

    fun convertDate(strDate: String): LocalDate {
        return LocalDate.parse(strDate, dataFormatter)
    }

    fun convertDateTime(strDate: String): LocalDateTime {
        return LocalDateTime.parse(strDate, dataTimeFormatter)
    }
